vello/src/util.rs

```diff
@@ -80,6 +80,52 @@ impl RenderContext {
         Ok(surface)
     }
 
+    /// Creates a new unsafe surface for the specified window and dimensions.
+    ///
+    /// # Safety
+    /// This function works exactly like `create_surface()`.
+    /// Treat it like `create_surface()` and you should not have any issues :D.
+    pub async unsafe fn create_surface_unsafe<'w>(
```

Comment:
Let's just expose a safe function which accepts a Surface, and document that `RenderSurface` drops the associated surface when it is dropped.

I.e.
```rust
pub async fn create_render_surface(&mut self, surface: wgpu::Surface<'w>, ...)->Result<RenderSurface<'w>>;
```
That way, we don't need to think about unsafety in the core Vello library.

The existing `create_surface` can be refactored in terms of this.

Reply:
[Something like this?](https://github.com/bella-project/vello/blob/309d86c703e8a96c5918cdeeaaa57e10fd49a049/vello/src/util.rs#L59)

[Usage in simple_sdl2.](https://github.com/bella-project/vello/blob/309d86c703e8a96c5918cdeeaaa57e10fd49a049/examples/simple_sdl2/src/main.rs#L34)